username_api.py

```diff
@@ -46,6 +46,11 @@ def get_avatar(website, username):
         if not result or not result[0]:
             return None
         result = result[0]
+    elif 'html_selector' in data:
+        soup = BeautifulSoup(response.text, 'html.parser')
+        img = soup.select(data['html_selector'])
```

Comment:
This is a list.  The variable should end with a `s`.  e.g. `images` or `selected_items`.